### ISSUE TITLE
SigmaDimensions not working

### ISSUE THREAD

**Issue by SuchIlluminati** [Sigma88/Sigma-Dimensions#78]
Hi, when I use Sigma Dimensions it doesn't work. When I play KSP it plays just fine but resizing/rescaling doesn't do anything, all the planets still have their default size. I tried it with the Rescale mod as well but that also didn't do anything. I'm running a clean install of KSP 1.6.1 (1.6.1.2401 to be exact) and Sigma Dimensions and Kopernicus are the only mods I have installed. Link to zip file with logs etc. [https://drive.google.com/file/d/1XWvbNj9aLcF6LB0IaHuK1EzXel74N2Ao/view?usp=sharing](url). Any help is greatly appreciated

**Comment by theonegalen** [comment]
At the very basic level, did you have modulemanager installed?

**Comment by SuchIlluminati** [comment]
> At the very basic level, did you have modulemanager installed?

Yes, I have module manager installed. I've included a screenshot of my gamedata folder in the zip file I've posted, if it helps.

**Comment by theonegalen** [comment]
I was using Sigma Dimensions recently, and it seemed to work with rescale 2.5. I did have a problem with it rescaling environmental visual enhancements clouds though.

**Comment by Sigma88** [comment]
@SuchIlluminati  the output log you posted is very short, it seems like the game didn't even start

there is this error at the beginning:

**Comment by Sigma88** [comment]
your settings file has an error:

![image](https://user-images.githubusercontent.com/12640834/51807854-8c880100-228c-11e9-9a48-7c0a724fcaa1.png)

you need to put Resize and Rescale on 2 different lines, otherwise the game won't recognize the setting

**Comment by SuchIlluminati** [comment]
> if this solves your issue feel free to close it
> 
> :)

Thanks, I'll give it a try
